keras/utils/conv_utils.py

```diff
@@ -146,13 +146,41 @@ def conv_input_length(output_length, filter_size, padding, stride):
     return (output_length - 1) * stride - 2 * pad + filter_size
 
 
-def deconv_length(dim_size, stride_size, kernel_size, padding):
+def deconv_length(dim_size, stride_size, kernel_size, padding, output_padding):
+    """Determines output length of a transposed convolution given input length.
+
+    # Arguments
+        dim_size: integer.
+        stride_size: integer.
+        kernel_size: integer.
```

Comment:
Please add quick descriptions for the arguments, and capitalize the first character after `: `